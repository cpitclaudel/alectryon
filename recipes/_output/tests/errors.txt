tests/errors.rst:18: (WARNING/2) Inline emphasis start-string without end-string.
tests/errors.rst:27: (ERROR/3) In :coqid:`A`: 'A' is not a fully-qualified name.
tests/errors.rst:30: (ERROR/3) In :coqid:`Lib.A`: Not sure where to find documentation for Lib.A.
Make sure that ‘Lib.A’ is fully qualified and that Alectryon knows where to find it.
Known prefixes: ['Coq']
(Add prefixes to alectryon.docutils.COQ_IDENT_DB_URLS or derive a new role from ‘coqid’ with a custom :url:).
tests/errors.rst:36: (ERROR/3) In :mref:`...`: Cannot parse marker-placement expression ``...``.
tests/errors.rst:37: (ERROR/3) In :mref:`.io`: Missing search pattern for key ``.io`` in expression ``.io``. (maybe an invalid pattern?)
tests/errors.rst:38: (ERROR/3) In :mref:`.abc`: Cannot parse marker-placement expression ``.abc``.
tests/errors.rst:39: (ERROR/3) In :mref:`.s()`: Cannot parse marker-placement expression ``.s()``.
tests/errors.rst:40: (ERROR/3) In :mref:`.s.g`: Missing search pattern for key ``.s`` in expression ``.s.g``. (maybe an invalid pattern?)
tests/errors.rst:41: (ERROR/3) In :mref:`.io{*}.g`: Unsupported search pattern {*} for key ``.io`` in expression ``.io{*}.g``.
tests/errors.rst:42: (ERROR/3) In :mref:`.s[a-z]`: Missing search pattern for key ``.s`` in expression ``.s[a-z]``. (maybe an invalid pattern?)
tests/errors.rst:43: (ERROR/3) In :mref:`.s(abc)[a-z]`: Cannot parse marker-placement expression ``[a-z]``.
tests/errors.rst:44: (ERROR/3) Error in "massert" directive:
Cannot parse marker-placement expression ``<.s(Goal)>``.

.. massert:: <.s(Goal)>

tests/errors.rst:47: (ERROR/3) In :mref:`.s(Goal).g#0.name`: ``.name`` is not supported in ``:mref:`` queries.
tests/errors.rst:50: (ERROR/3) In :mref:`.g#0.in`: Missing .s(…) sentence component in path.
tests/errors.rst:53: (ERROR/3) In :mref:`.s(Goal).in.ccl`: Incompatible components in path ``.s(Goal).in.ccl``: not sure what to do with ``in``.
tests/errors.rst:56: (ERROR/3) In :mquote:`.s(Goal)`: Cannot quote a full sentence (``.s``) inline.
tests/errors.rst:57: (ERROR/3) In :mquote:`.s(Goal).g#1`: Cannot quote a full goal (``.g``) inline.
tests/errors.rst:58: (ERROR/3) Error in "mquote" directive:
Cannot quote a full sentence (``.s``) inline.

.. mquote:: .s(Goal)
tests/errors.rst:59: (ERROR/3) Error in "mquote" directive:
Cannot quote a full goal (``.g``) inline.

.. mquote:: .s(Goal).g#1

tests/errors.rst:62: (ERROR/3) In :mquote:`test <.s(Goal)>`: Title syntax (``… <…>``) not supported with ``:mquote:``.
tests/errors.rst:63: (ERROR/3) Error in "mquote" directive:
Title syntax (``… <…>``) not supported with ``:mquote:``.

.. mquote:: test <.s(Goal)>
tests/errors.rst:64: (ERROR/3) Error in "massert" directive:
Title syntax (``… <…>``) not supported with ``:massert:``.

.. massert::

   test <.s(Goal)>

tests/errors.rst:69: (ERROR/3) Error in "role" directive:
invalid option value: (option: "prefix"; value: None)
argument required but none supplied.

.. role:: mq2(mquote)
   :prefix:
tests/errors.rst:71: (ERROR/3) Error in "role" directive:
invalid option value: (option: "prefix"; value: '.s.i')
Missing search pattern for key ``.s`` in expression ``.s.i``. (maybe an invalid pattern?).

.. role:: mq2(mquote)
   :prefix: .s.i

tests/errors.rst:107: (ERROR/3) Error in "coq" directive:
Unknown flag `unknown`.

.. coq:: unknown

   Check nat.

tests/errors.rst:112: (ERROR/3) Error in "coq" directive:
Unrecognized directive flags: .xyz

.. coq:: unfold out .xyz

   Check nat.

tests/errors.rst:117: (ERROR/3) Error in "coq" directive:
Missing search pattern for key ``.s`` in expression ``.s.g``. (maybe an invalid pattern?)

.. coq:: .s.g

   Check nat.

tests/errors.rst:5: (WARNING/2) In `:alectryon/pygments/tacn:`: Missing language name (did you mean `:alectryon/pygments/coq/tacn:`?).
tests/errors.rst:6: (ERROR/3) In `:alectryon/pygments/coq/xyz:`: Unknown token kind: xyz
tests/errors.rst:146: (ERROR/3) Coq raised an exception:
  > Syntax error: '.' expected after [syntax_command] (in [vernac_aux]).
The offending chunk is delimited by >>>…<<< below:
  > Notation "'🆄🄽🅘ⓒ⒪𝖉∈' α ⊕ ⤋ β" := α >>>++<<< β. (* Bytes vs. str *)
Results past this point may be unreliable.
tests/errors.rst:151: (ERROR/3) Coq raised an exception:
  > The term "true" has type "bool" which should be Set, Prop or Type.
The offending chunk is delimited by >>>…<<< below:
  > Definition a
  >   : >>>true<<< := 1. (* Next line *)
Results past this point may be unreliable.
tests/errors.rst:77: (WARNING/2) Long line (107 characters)
     exact I. (* A very long line *) (* whose contents are split *) (* across *) (* multiple *) (* comments *)
tests/errors.rst:122: (ERROR/3) In `.. coq::`:
   Missing search pattern for key ``.io`` in expression ``.io.s``. (maybe an invalid pattern?)
   Unknown flag `xyz`.
tests/errors.rst:128: (ERROR/3) In `.. coq::`:
   ``.io`` not supported in visibility annotations
   ``name`` not supported in visibility annotations.
   ``body`` not supported in visibility annotations.
   No match found for `.g#1` in `Check nat.`
   No match found for `.msg{*}` in `Abort.`
tests/errors.rst:137: (ERROR/3) In `.. coq:: unfold out`: Cannot show output of 'Check nat.' without .in or .unfold.
tests/errors.rst:75: (ERROR/3) In :mref:`.s(Goal True)`: Not sure which code block this refers to; add ``.io#…`` to disambiguate.
tests/errors.rst:85: (ERROR/3) In :mref:`.io#nope.s(123)`: Reference to unknown .io block nope. 
tests/errors.rst:86: (ERROR/3) In :mref:`.s(Goal).g#25`: No goal matches '25'.
tests/errors.rst:87: (ERROR/3) In :mref:`.s(pose proof).h#n.body`: Target is null
tests/errors.rst:92: (ERROR/3) In assertion `.g{*not found*}`: No goal matches '*not found*'.
tests/errors.rst:94: (ERROR/3) In assertion `.msg`: No message matches ''.
exit: 13
