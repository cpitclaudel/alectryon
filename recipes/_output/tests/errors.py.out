test_errors (__main__.cli.test_errors) ... ok
test_warnings_and_errors (__main__.coqc_time.test_warnings_and_errors) ... ok
test_errors (__main__.core.test_errors) ... ok
test_errors (__main__.docutils.test_errors) ... ok
test_errors (__main__.json.test_errors) ... ok
test_warnings (__main__.json.test_warnings) ... ok
test_errors (__main__.literate.test_errors) ... ok
test_features (__main__.literate.test_features) ... ok
test_failed_import (__main__.myst.test_failed_import) ... ok
test_errors (__main__.pygments.test_errors) ... ok
test_warnings (__main__.pygments.test_warnings) ... ok
test_warnings_and_errors (__main__.serapi.test_warnings_and_errors) ... ok
test_errors (__main__.sexp.test_errors) ... ok

----------------------------------------------------------------------
Ran 13 tests

OK
